hotsos/core/plugins/kernel/memory.py

```diff
@@ -37,11 +42,15 @@ def __getattr__(self, key):
                     self._proc_file_keys_values[key] = value
                     return value
 
+        if key in self.VALID_KEYS:
+            return self.DEFAULT_RETURN_VALUE
```

Comment:
nit: I guess this could just be 'return 0' as is done above on L36 since sub-classes don't seem to override it anywhere.

Reply:
They don't currently override it but i thought should make it possible to do so since different files targets might want a different default. In reality they will probably all want 0 though yeah.